server/svix-server/src/v1/endpoints/message.rs

```diff
@@ -204,7 +212,7 @@ async fn create_message(
         Some(app.clone()),
         app.id.clone(),
         app.org_id,
-        Duration::from_secs(30),
+        Duration::seconds(30).to_std().unwrap(),
```

Comment:
Why was this change needed?

Reply:
I pulled in chrono::Duration, replacing std::time::Duration

Reply:
```suggestion
        std::time::Duration::from_secs(30),
```